src/components/Footer.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import Responsive from 'react-responsive-decorator';
+import './style.css';
+
+class Footer extends Component {
+    state = {}
+
+    render() { 
+        return (
+            <section>
+                <div className="footer">
+                    <div className="col-md-12">
+                        <div className="row">
+                            <div className="col-md-9 text-left">
+                                <p>Hosted by: <a href="http://www.seti.org">The SETI Institute</a></p>
```

Comment:
We can probably add `target` for those href tag

```suggestion
                                <p>Hosted by: <a href="http://www.seti.org" target="_blank">The SETI Institute</a></p>
```

So that a new tab is opened if the user clicks on the link

Reply:
Noted